resources/views/components/game/compare-progress.blade.php

```diff
@@ -86,27 +93,40 @@ function selectSearchBoxUser() {
             <p>None of your followed users have played this game</p>
         @else
             <p>Compare to a followed user:</p>
-            <table class="table-highlight"><tbody>
-            @foreach ($followedUserCompletion as $completion)
-                <?php $friend = User::find($completion['user_id']); ?>
-                <tr>
-                    <td>{!! userAvatar($friend, iconSize: 20) !!}</td>
-                    <td class="text-right">
-                        <a href="{!! route('game.compare-unlocks', ['game' => $game, 'user' => $friend->User]) !!}">
-                            {{ $completion['achievements_unlocked'] }}/{{ $completion['achievements_total'] }}
-                        </a>
-                    </td>
-                </tr>
-            @endforeach
-            </tbody></table>
+            <div class="">
+                @foreach ($followedUserCompletion as $completion)
+                    @php
+                        $friend = User::find($completion['user_id']);
+                    @endphp
+                    <div class="odd:bg-embed hover:bg-embed-highlight border border-transparent hover:border-[rgba(128,128,128,.3)] p-1 flex items-center justify-between w-full">
+                        <div>
+                            {!! userAvatar($friend, iconSize: 20) !!}
+                        </div>
 
+                        <a
+                            class="min-w-[100px] flex flex-col"
+                            href="{!! route('game.compare-unlocks', ['game' => $game, 'user' => $friend->User]) !!}"
+                        >
+                            <x-game-progress-bar
+                                :softcoreProgress="$completion['achievements_unlocked']"
+                                :hardcoreProgress="$completion['achievements_unlocked_hardcore']"
+                                :maxProgress="$completion['achievements_total']"
+                            />
+                            <p class="text-2xs">
+                                {{ $completion['achievements_unlocked']}} of {{ $completion['achievements_total'] }}
+                            </p>
```

Comment:
I think it might be better without the sublabel. The tooltip only appears over the bar, and the label makes it hard to hover over the bar. The tooltip also shows the same information, but includes the hardcore/softcore breakdown. Not having this sublabel would also be more consistent with the same widget's use in the recent players list.

Reply:
Label removed. Added a way of increasing padding around the progress bar container so the tooltip is easier to surface.